metrics-common/src/main/java/org/apache/helix/monitoring/mbeans/dynamicMBeans/DynamicMBeanProvider.java

```diff
@@ -227,6 +227,14 @@ protected void incrementSimpleDynamicMetric(SimpleDynamicMetric<Long> metric, lo
     metric.updateValue(metric.getValue() + value);
   }
 
+  /**
+   * NOTE: This method is not thread-safe nor atomic.
+   * Decrement the value of a given SimpleDynamicMetric with input value.
+   */
+  protected void decrementSimpleDynamicMetric(SimpleDynamicMetric<Long> metric, long value) {
```

Comment:
NIT: Is this necessary? We can increase with negative value so minimize the APIs.